pkg/slo-controller/noderesource/noderesource_controller_test.go

```diff
@@ -18,8 +18,8 @@ package noderesource
 
 import (
 	"context"
+
```

Comment:
no need to add a new line between `context` and `testing`

Reply:
nice catch. fixed